src/modules/job-exec/job-exec.c

```diff
@@ -696,7 +696,8 @@ static flux_future_t * namespace_move (struct jobinfo *job)
 {
```

Comment:
In first line of commit description in `fix potential namespace leak`, there is a stray `a`.

Reply:
Thanks! Fixed and will set MWP.